ai-platform-unified/notebooks/unofficial/pipelines/automl_tabular_classification_beans.ipynb

```diff
@@ -0,0 +1,1049 @@
+{
```

Comment:
<p>Can you use the format in the notebook_template. It uses a IS_GOOGLE_CLOUD_NOTEBOOK variable.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/367/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
done